player/command.c

```diff
@@ -7594,6 +7602,18 @@ static void command_event(struct MPContext *mpctx, int event, void *arg)
         run_command_opts(mpctx);
     }
 
+    if (event == MPV_EVENT_VIDEO_RECONFIG && ctx->hwdec_osd_mode) {
+        struct track *track = mpctx->current_track[0][STREAM_VIDEO];
+        struct mp_decoder_wrapper *dec = track ? track->dec : NULL;
+        char *current = NULL;
+        if (dec)
+            mp_decoder_wrapper_control(dec, VDCTRL_GET_HWDEC, &current);
+        if (!current || !current[0])
+            current = "no";
```

Comment:
Where is this `current` used?

Reply:
I'm dumb. None of this junk is even needed.